t/t5505-remote.sh

```diff
@@ -924,7 +925,8 @@ test_expect_success 'migrate a remote from named file in $GIT_DIR/branches (2)'
 	(
 		cd seven &&
 		git remote rm origin &&
-		echo "quux#foom" > .git/branches/origin &&
```

Comment:
White space change around the `>`. We should either change all of them or leave as a separate tidy up.

Reply:
Right.

Reply:
Extracted into its own commit.